File: Angular_UIL/ClientApp/src/app/models/Transactions/StoreTransaction/SaveStoreTransaction/StoreTransDetails/ItemDetails_VM.ts
import { Items_VM } from "./Items_VM";
import { Unites_VM } from "./Unites_VM";

export class ItemDetails_VM {
  constructor(
    public items_VM?: Items_VM[],
    public unites_VM?: Unites_VM[],
    public Price?: number,
  ) {

  }
}

File: BLL/ModelsView/ItemDetails_VM.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Text;
using System.Threading.Tasks;

namespace BLL.ModelsView
{
   public class ItemDetails_VM
    {
        public ItemDetails_VM()
        {
            Items_VMs = new List<Items_VM>();
            Unites_VMs = new List<Unites_VM>();


        }
        public List<Items_VM> Items_VMs { get; set; }
        public List<Unites_VM> Unites_VMs { get; set; }
        public int Price { get; set; }
    }
}
